v2/internal/pkg/release/signature.go

```diff
@@ -169,14 +169,19 @@ func (o SignatureSchema) GenerateReleaseSignatures(ctx context.Context, images [
 			}
 			img.Source = signSchema.Critical.Identity.DockerReference
 			o.Log.Debug("image found : %s", signSchema.Critical.Identity.DockerReference)
-			// o.Log.Info("public Key : %s", strings.ToUpper(fmt.Sprintf("%x", md.SignedBy.PublicKey.Fingerprint)))
-
 			// write signature to cache
 			ferr := os.WriteFile(o.Opts.Global.WorkingDir+SignatureDir+digest, data, 0644)
 			if ferr != nil {
 				o.Log.Error("%v", ferr)
 			}
 			imgs = append(imgs, img)
+
+			cr := clusterresources.New(o.Log, o.Opts.Global.WorkingDir, o.Config, "")
+			err = cr.GenerateSignatureConfigMap(digest, id, data)
+			if err != nil {
+				o.Log.Error("%v", err)
+			}
+
```

Comment:
Currently all the clusterresources are being generated in the executor.go, the generation of the SignaturesConfigMap could follow the same standard.

Here is an example: https://github.com/openshift/oc-mirror/blob/8d037bac34651ea92e52f46757854d1b53b8ebac/v2/internal/pkg/cli/executor.go#L781

Having the generation of the SignatureConfigMap in the executor could bring some benefits:

* The SignatureConfigMap will be generated only if the workflow finished successfully (in the collector it will be always generated)
* If the customer is running a dry run, having it in the executor.go will avoid the generation of the SignatureConfigMap (in the collector it will be always generated)
* Having the clusterresources being generated at only one place is easy to maintain the code in the long term
* Since the executor already has the dependency on the clusterresources pkg it is possible to avoid an additional dependency of the clusterresources pkg in the collector pkg
* Having the release collector calling the clusterresources (write results) pkg directly will break the architecture defined here: https://github.com/openshift/oc-mirror/blob/main/v2/assets/architecture.png where the executor should be the responsible to call the clusterresources (write results)